convert2rhel/unit_tests/subscription_test.py

```diff
@@ -1407,6 +1406,10 @@ def test_lock_releasever_in_rhel_repositories(pretend_os, subprocess, expected,
         "run_subprocess",
         value=run_subprocess_mock,
     )
+
+    major_version = EUS_MINOR_VERSIONS[0][0]
+    minor_version = EUS_MINOR_VERSIONS[0][2]
+    monkeypatch.setattr(system_info, "version", namedtuple("Version", ["major", "minor"])(major_version, minor_version))
```

Comment:
I think you could also do something like: 

```suggestion
    eus_version = EUS_MINOR_VERSION[0].split(".")
    major_version = eus_version[0]
    minor_version = eus_version[1]
    monkeypatch.setattr(system_info, "version", namedtuple("Version", ["major", "minor"])(major_version, minor_version))
```

Would look a bit cleaner and easier to understand. No need to change right now, just writing down the suggestion.

Reply:
It does look better! I can add that in, wondering if we should wait till the first version is merged or not.

Reply:
Since everything passed, I would say to merge this first and then later do this change. No worries.